MaterialDesignThemes.UITests/Samples/PasswordBox/BoundPasswordBox.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<UserControl x:Class="MaterialDesignThemes.UITests.Samples.PasswordBox.BoundPasswordBox"
```

Comment:
This control and the window and view model residing in the folder are simple here to be able to reproduce the non-working scenarios in UI tests.